components/media_player/ui/Card.qml

```diff
@@ -311,7 +310,7 @@ Rectangle {
                 return i;
             }
             //anchors { bottom: parent.bottom; bottomMargin: 20; horizontalCenter: parent.horizontalCenter }
-            anchors { bottomMargin: 20; horizontalCenter: bottomMenuScrollArea.horizontalCenter }
+            anchors { centerIn: parent; bottomMargin: 20; horizontalCenter: bottomMenuScrollArea.horizontalCenter }
```

Comment:
`centerIn` centers the object inside parten's center. `horizontalCenter` is not needed here because `centerIn` does horizontal and vertical centering. Also please make sure to use the same reference to objects. `parent` and `bottomMenuScrollArea` is the same here.

Reply:
done